controllers/perconaserverformysql_controller.go

```diff
@@ -243,48 +270,57 @@ func (r *PerconaServerForMySQLReconciler) reconcileReplication(
 
 func reconcileReplicationPrimaryPod(
 	ctx context.Context,
-	cl k8s.APIListPatcher,
+	cl client.Client,
 	cr *apiv2.PerconaServerForMySQL,
 ) error {
+	l := log.FromContext(ctx).WithName("reconcileReplicationPrimaryPod")
+
 	pods, err := k8s.PodsByLabels(ctx, cl, mysql.MatchLabels(cr))
 	if err != nil {
 		return errors.Wrap(err, "get MySQL pod list")
 	}
+	l.Info(fmt.Sprintf("got %v pods", len(pods)))
 
 	host := orchestrator.APIHost(orchestrator.ServiceName(cr))
 	primary, err := orchestrator.ClusterPrimary(ctx, host, cr.ClusterHint())
 	if err != nil {
-		return errors.Wrap(err, "get cluster from orchestrator")
+		return errors.Wrap(err, "get cluster primary")
 	}
 	primaryAlias := primary.Alias()
+	l.Info(fmt.Sprintf("got cluster primary alias: %v", primaryAlias), "data", primary)
```

Comment:
It seems like a debug log

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/22#discussion_r767727670